src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -20,29 +20,39 @@ import { createAndLogError, logger } from "../../utils/logger";
 import { toReadableFormat } from "../../utils/date";
 import { sendUpdate } from "../../utils/update.confirmation.statement.submission";
 import { formatPSCForDisplay, formatAddressForDisplay } from "../../utils/format";
+import { FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY } from "../../utils/properties";
```

Comment:
The flag has been renamed to include the _date on it, might want to pull main into this and update the flag as this wont work when it goes into main otherwise